modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -52,7 +51,8 @@ export class Utils implements BaseUtils {
 
   /** @inheritdoc */
   isValidSignature(signature: string): boolean {
-    throw new NotImplementedError('method not implemented');
+    const signatureU8a = u8aToU8a(signature);
+    return [64, 65, 66].includes(signatureU8a.length);
```

Comment:
This is pretty much what `signatureVerify()` does in the polkadot lib

Reply:
Yeah - does it make sense to just have it use it like this:
`const { isValid } = signatureVerify(message, signature, alice.address);`

source: https://polkadot.js.org/docs/keyring/start/sign-verify/

Reply:
Oh, we have that method right below here `verifySignature(signedMessage: string, signature: string, address: string): boolean`